sql/pipelines/src/test/scala/org/apache/spark/sql/pipelines/graph/MaterializeTablesSuite.scala

```diff
@@ -33,10 +33,10 @@ import org.apache.spark.util.Utils.exceptionString
  * tables are written with the appropriate schemas.
  */
 class MaterializeTablesSuite extends BaseCoreExecutionTest {
-
-  import originalSpark.implicits._
-
   test("basic") {
+    val session = spark
+    import session.implicits._
```

Comment:
do we still need these changes after using `SharedSparkSession`?

Reply:
It appears that we still do

Reply:
Cool, I will merge it to unblock the build failure